receiver/hostmetricsreceiver/hostmetrics_receiver_test.go

```diff
@@ -47,6 +47,7 @@ var standardMetrics = []string{
 	"system.memory.usage",
 	"system.disk.io",
 	"system.disk.ops",
+	"system.disk.pending_operations",
```

Comment:
Should we unabbreviate ops in other metrics?

Reply:
Yes the OTEP recently changed these to long names. I will update names again in a follow up PR